src/main/scala/scynamo/syntax/AttributeValueDslOps.scala

```diff
@@ -1,19 +1,21 @@
-package scynamo.syntax.attributevalue
+package scynamo.syntax
 
-import java.util
-
-import cats.syntax.either._
 import cats.data.NonEmptyChain
+import cats.syntax.either._
 import scynamo.ScynamoDecodeError.TypeMismatch
 import scynamo.{ScynamoDecodeError, ScynamoType}
 import software.amazon.awssdk.core.SdkBytes
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue
 
+import java.util
+
+object attributevalue extends AttributeValueDsl
+
 trait AttributeValueDsl {
   implicit def toAttributeValueOps(attributeValue: AttributeValue): AttributeValueDslOps = new AttributeValueDslOps(attributeValue)
 }
 
-class AttributeValueDslOps(val attributeValue: AttributeValue) extends AnyVal {
+class AttributeValueDslOps(val attributeValue: AttributeValue) extends AttributeValueDsl {
```

Comment:
Why this change?

Reply:
Because `asEither` in line 45 below uses the `asOption` and needs the implicit conversion.

Reply:
Yeah but can't we just import it there? This is really weird and less efficient.

Reply:
Like in https://github.com/moia-oss/scynamo/pull/232/commits/bc12161bfef9d173cd6097fd369111f210dcf904?